sirepo/pkcli/job_agent.py

```diff
@@ -775,7 +766,7 @@ def _processor():
 #SBATCH --constraint={_processor()}
 #SBATCH --qos={self.msg.sbatchQueue}
 #SBATCH --tasks-per-node={self.msg.tasksPerNode}
-{_assert_project()}"""
+{sirepo.nersc.assert_project(self.msg.sbatchProject)}"""
```

Comment:
assert_project doesn't tell the whole story. Now that it is a public function it should say what it does: sbatch_project_option()

Reply:
[ee288f3](https://github.com/radiasoft/sirepo/pull/5493/commits/ee288f301b4e638df09ad32e740bbb3566224860)